src/organisms/PageMap.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { Theme } from '@/theme/Theme';
+import { Map, MapContainer } from '@/atoms/MapAtoms';
+import { MapLayers } from '@/molecules/MapLayers';
+import { TogglePanel } from '@/molecules/TogglePanel';
+
+export const ExplorePageMap = ({ data, featureToggleStates }: { data: MapGeoJsonData[], featureToggleStates: FeatureToggleState }) => {
```

Comment:
There's a template (as per atomic design) in here that can be extracted out... But best not to over-abstract until we see the need to use it